packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraXProxy.java

```diff
@@ -48,4 +50,15 @@ public SystemServicesFlutterApiImpl createSystemServicesFlutterApiImpl(
       @NonNull BinaryMessenger binaryMessenger) {
     return new SystemServicesFlutterApiImpl(binaryMessenger);
   }
+
+  public ImageCapture.Builder createImageCaptureBuilder() {
```

Comment:
Why make this a method instead of just calling ImageCapture.Builder() were you need it?

Reply:
For injecting a mock for testing